.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,11 @@
-<!-- Please take a look at our Contributing documentation before submitting a Pull Request!
-https://rook.io/docs/rook/latest/Contributing/development-flow/
+<!-- Thank you for contributing to Rook! -->
 
-Thank you for contributing to Rook! -->
-
-**Description of your changes:**
+<!-- STEPS TO FOLLOW:
+  1. Add a description of the changes (frequently the same as the commit description)
```

Comment:
I would see the commit section linked here([Commit-Structure](https://rook.io/docs/rook/latest-release/Contributing/development-flow/#commit-structure)), as many times I have seen commit-lint failing.

Reply:
I revised step 4 to be clear about the commit message. Does that help?